### PR TITLE
Update to monaco-vscode-api 3.2.1

### PR DESCRIPTION
**Pull request by kaisalmen**
Using the standalone packages from monaco-editor works. Transformation of workers to esm bundles also works fine. Loading the workers directly via the factory function does not. I don't understand yet why. Editor worker is no problem.

### CODE REVIEW

**Comment by CGNonofr** [comment]
What is the errors with the workers?

**Comment by kaisalmen** [comment]
First, one guy called Kai is unable to type properly. Then vite gets confused if you specify this (js extension): 
```js
typescript: () => new Worker(new URL('@codingame/monaco-vscode-standalone-typescript-language-features/worker.js', import.meta.url), { type: 'module' })
```
instead of this:
```js
typescript: () => new Worker(new URL('@codingame/monaco-vscode-standalone-typescript-language-features/worker', import.meta.url), { type: 'module' })
```

**Comment by CGNonofr** [comment]
> @CGNonofr first, one guy called Kai is unable to type properly. Then vite gets confused if you specify this (js extension):
> 
> ```js
> typescript: () => new Worker(new URL('@codingame/monaco-vscode-standalone-typescript-language-features/worker.js', import.meta.url), { type: 'module' })
> ```
> 
> instead of this:
> 
> ```js
> typescript: () => new Worker(new URL('@codingame/monaco-vscode-standalone-typescript-language-features/worker', import.meta.url), { type: 'module' })
> ```

Yep, that's because as soon as you have an `exports` field in your package.json, you can't import any other file directly. It's not really a vite specificity. What is specific to vite is that if the url of the worker can't be resolved, it just gives up transforming it without errors.

So everything is fine now?

**Comment by kaisalmen** [comment]
> What is specific to vite is that if the url of the worker can't be resolved, it just gives up transforming it without errors.

It says the file is not there and that is just not true. It has no visibility due to the extra export. I like that you did that actually.

> So everything is fine now?

Yes

**Comment by kaisalmen** [comment]
@CGNonofr I disabled the yarn verification. It is making trouble all the time because it can't resolve local dependencies.

File: README.md [hunk 1]
```diff
@@ -251,7 +251,7 @@ loader.config({ monaco });
 If you use pnpm, you have to add `vscode` / `@codingame/monaco-vscode-api` as direct dependency (you find the [compatibility table here](https://github.com/TypeFox/monaco-languageclient/blob/main/docs/versions-and-history.md#monaco-editor--codingamemonaco-vscode-api-compatibility-table), otherwise the installation will fail.
 
 ```json
-"vscode": "npm:@codingame/monaco-vscode-api@~3.1.2"
+"vscode": "npm:@codingame/monaco-vscode-api@"
```
Comment (CGNonofr) [inline README.md:254]: does it work with a trailing @ ?

File: packages/client/CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,10 @@
 
 All notable changes to this npm module are documented in this file.
 
+## [8.0.1] - 2024-03-2x
+
+- Blah
```
Comment (CGNonofr) [inline packages/client/CHANGELOG.md:7]: Blah blahblah? blah blah!

File: README.md (hunk 1 above)
Comment (kaisalmen) [reply README.md]: That was search/replace error.

File: packages/client/CHANGELOG.md (hunk 2 above)
Comment (kaisalmen) [reply packages/client/CHANGELOG.md]: Yeah, that was not really finished. 😊 I added entries in all CHANGELOGs that required adjustment and also fixed some minor errors in the READMEs (example links were wrong).

**Comment by kaisalmen** [comment]
@CGNonofr this is now really ready.

**Review by CGNonofr — APPROVED** [review]

**Pull request merged**